canonicalwebteam/discourse/parsers/base_parsers.py

```diff
@@ -0,0 +1,223 @@
+# Standard library
+import re
+
+# Packages
+from bs4 import BeautifulSoup
+from urllib.parse import urlparse
+
+
+TOPIC_URL_MATCH = re.compile(
+    r"(?:/t)?(?:/(?P<slug>[^/]+))?/(?P<topic_id>\d+)(?:/\d+)?"
```

Comment:
I always find it helpful when people write regex's to add a description of what this matches and does not match so that you don't have to read to far into the regex unless you're debugging something. It also helps for people who aren't fluent in regex to understand what's happening.

I also don't see tests for this? Are they somewhere else?

Reply:
Thanks. Added comment and test